plugins/shared/info-panel/error.handlebars

```diff
@@ -11,5 +11,10 @@
             {{{title}}}
         </div>
     </a>
-    <div class="tota11y-info-error-description tota11y-collapsed"></div>
+    <div class="tota11y-info-error-description tota11y-collapsed">
+        <div class="tota11y-info-error-description-code-container"> 
+            Element's Code:
+            <code></code> 
```

Comment:
Oh sorry, my previous comment was asking to also add a class here, and update the Less instead of selecting all `code` elements.